bindings/nodejs/lib/types/models/api/committee-response.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { Bech32Address, EpochIndex } from '../../block';
+import { u64 } from '../../utils';
+
+/**
+ * Returns information of a validator (committee member).
+ */
+export interface CommitteeMember {
+    /**
+     * Account address of the validator.
+     */
+    address: Bech32Address;
+    /**
+     * The total stake of the pool, including delegators.
+     */
+    poolStake: u64;
```

Comment:
I think @Thoralf-M said `u64` doesn't work in interfaces?

Reply:
why is every other language than Rust so weird???

Reply:
Ah yeah should be NumericString